### PR TITLE
WASM: Minor Improvement

### PR DESCRIPTION
**Pull request by Shaikh-Ubaid**
fixes https://fortran-lang.discourse.group/t/lfortran-installation-problem/5439
Also towards https://github.com/lcompilers/lpython/issues/1485

### CODE REVIEW

**Comment by Shaikh-Ubaid** [comment]
This is ready.

File: ci/environment.yml [hunk 1]
```diff
@@ -6,6 +6,7 @@ dependencies:
   - toml
   - pytest
   - jupyter
+  - nbconvert!=7.3.0
```
Comment (czgdp1807) [inline ci/environment.yml:9]: This belongs here or in https://github.com/lfortran/lfortran/pull/1496?

**Review by czgdp1807 — APPROVED** [review]

**Comment by certik** [comment]
@Shaikh-Ubaid I clicked on "update with rebase", but I think GitHub didn't do this correctly, it included commits that are already merged in master. Do you please mind manually rebasing this? Sorry about it.

File: ci/environment.yml (hunk 1 above)
Comment (Shaikh-Ubaid) [reply ci/environment.yml:9]: The PR needs a rebase. That commit in this PR is more to see if the CI Passes.

**Review by certik — APPROVED** [review]
The changes are fine, but the PR needs a manual rebase. It shouldn't have any changes in the environment.yml.

**Comment by Shaikh-Ubaid** [comment]
>The changes are fine, but the PR needs a manual rebase. It shouldn't have any changes in the environment.yml.

Got it. Done. After the rebase, there are no changes in `environment.yml`.

As this PR seems approved, I am adding it to auto-merge.

**Pull request merged**